packages/content-conformance/src/rules/must-have-h1.js

```diff
@@ -0,0 +1,32 @@
+// eslint-disable-next-line import/no-anonymous-default-export
+/** @type {import('../types.js').ConformanceRuleBase} */
+export default {
+  type: 'content',
+  id: 'must-have-h1',
+  description: 'Each content file must contain an H1 the top.',
+  executor: {
+    async contentFile(file, context) {
+      let hasH1 = false
+      let hasVisitedAHeading = false
+
+      file.visit(['heading'], (node) => {
+        hasVisitedAHeading = true
+
+        if (node.depth === 1) {
+          hasH1 = true
+        }
+
+        if (!hasH1 && hasVisitedAHeading) {
+          context.report(
+            'Must have a level 1 heading at the top of the file.',
```

Comment:
[suggestion]: Some additional messaging here would be hugely valuable and go a long way for end users who need to take action to resolve these errors, and who likely do not have the same context that we rule-authors have.

Something like:
- what @BRKalow did [here](https://github.com/hashicorp/web-platform-packages/pull/142/files/eed0655f2da273cac15bdcd95d715ec620caf079#diff-0a6b4349881264f7698ab156c0f077160ce35ac83803f26f404b6e7b5ad520f5R116-R119) 
- or what I did [here](https://github.com/hashicorp/web-platform-packages/pull/137/files#diff-60bdaf8264d1260790c7fe018f88599c67ce11cef4aab81b44aadeae8c01125bR24-R30)

Inspiration: https://vercel.com/design/error#error-design-framework

Reply:
Thanks!